components/widgets/admin_console/admin_panel_with_button.tsx

```diff
@@ -26,7 +26,7 @@ const AdminPanelWithButton: React.FC<Props> = (props: Props) => {
         const buttonId = (props.buttonTextDefault || '').split(' ').join('-').toLowerCase();
         button = (
             <a
-                className='btn btn-primary'
+                className={props.disabled ? 'btn btn-primary disabled' : 'btn btn-primary'}
```

Comment:
```suggestion
                className={classNames('btn', 'btn-primary', {'disabled': props.disabled})}
```

with an `import classNames from 'classnames';` at the top

Reply:
Good idea. Thanks.